packages/kit/src/core/prerender/prerender.js

```diff
@@ -136,9 +136,9 @@ export async function prerender({ config, entries, files, log }) {
 			}
 		});
 
-		const text = await response.text();
+		const body = Buffer.from(await response.arrayBuffer());
```

Comment:
actually, i guess not, since `Buffer.from(...).toString()` does something useful unlike `new Uint8Array(...).toString()`. But the type signature for `save` expects a `string | Uint8Array`, and we're creating a `Uint8Array` on line 149.

I feel like we need some more certainty here — we should probably have some tests as well

Reply:
I think always using buffer is not a solution either, I've tried that and if there is no buffer, it stops whole thing from crawling any further without any error. So there is check needed whether it can be converted to buffer.

Reply:
TIL `new Buffer(0) instanceof Uint8Array`, which is why TypeScript didn't complain about this